lib/LanguageServerCodeTransform/Model/NameImport/NameImporter.php

```diff
@@ -64,12 +66,29 @@ public function __invoke(
                 $prefix = $name->toArray()[0];
             }
 
-            return $this->__invoke($document, $offset, $type, $fqn, $prefix . $error->name());
+            return $this->__invoke($document, $offset, $type, $fqn, $updateReferences, $prefix . $error->name());
         } catch (AliasAlreadyUsedException $error) {
             $prefix = 'Aliased';
-            return $this->__invoke($document, $offset, $type, $fqn, $prefix . $error->name());
+            return $this->__invoke($document, $offset, $type, $fqn, $updateReferences, $prefix . $error->name());
         } catch (TransformException $error) {
             return NameImporterResult::createErrorResult($error);
         }
     }
+
+    private function createResultForAlreadyImportedFQN(
+        NameImport $nameImport,
+        NameAlreadyImportedException $error
+    ): NameImporterResult {
+        if ($error->existingName() !== $nameImport->name()->head()->__toString()) {
+            $alias = $error->existingName();
+        } else {
+            $alias = null;
```

Comment:
could remove the else and iniitalize `$alias` as `null`

Reply:
Done